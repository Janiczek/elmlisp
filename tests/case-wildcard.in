(case msg
  (NoOp 1)
  (_ 2))
