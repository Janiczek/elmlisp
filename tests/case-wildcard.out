case msg of
    NoOp ->
        1

    _ ->
        2
